src/main/java/duke/calendar/Date.java

```diff
@@ -0,0 +1,147 @@
+package duke.calendar;
+
+import duke.exception.DukeException;
+
+import java.time.Month;
+
+/**
+ * Represents a date. A <code>Date</code> object corresponds to a time represented by a day, month and year.
+ */
+public class Date {
```

Comment:
Interesting that you used your own Date class, but perhaps you can use Date from java.util.Date instead.